couchdb/src/test/java/org/hibernate/ogm/datastore/couchdb/test/crud/EntityWithByteArrayTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.ogm.datastore.couchdb.test.crud;
+
+import java.util.Arrays;
+import java.util.Random;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.ogm.utils.OgmTestCase;
+import org.hibernate.ogm.utils.TestForIssue;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertNotNull;
+
+/**
+ * @author Hardy Ferentschik
+ */
+public class EntityWithByteArrayTest extends OgmTestCase {
```

Comment:
> Ah yes, we should create a dedicated issue for supporting byte[] without a @Lob for all datastores. I did not think that was supported and indeed JPA's spec is not clear. But Hibernate ORM itself supports byte[] without @Lob.

+1 for creating a more general issue for supporting this on all data stores, provided we agree that it is valid. I goes if we decided to make it not valid, it would be problematic anyways, right? One would in this case expect some sort of mapping exception, but the configuration (annotation parsing etc) is done in ORM code, right? There is no simple way for us to change/modify that?